.github/workflows/main.yml

```diff
@@ -100,12 +101,9 @@ jobs:
         if: runner.os != 'Windows'
         run: ${{ runner.os == 'macOS' && 'ci/cirrusci.sh' || 'sudo -E ci/cirrusci.sh' }}
 
-      # NOTE: Linker ICEs with Xcode 15.0.1 (default version on macos-13)
-      # * https://issues.dlang.org/show_bug.cgi?id=24407
-      # Remove this step if the default gets changed to 15.1 in actions/runner-images.
-      - name: 'macOS 13: Switch to Xcode v15.1'
-        if: matrix.os == 'macos-13'
-        run: sudo xcode-select -switch /Applications/Xcode_15.1.app
```

Comment:
The current default Xcode version for macos-13 is v15.2, fine with the non-bootstrap DMD host compiler.